services/worker/src/worker/job_runners/split/descriptive_statistics.py

```diff
@@ -733,13 +735,14 @@ def compute_descriptive_statistics_response(
     def _column_from_feature(
         dataset_feature_name: str, dataset_feature: Union[dict[str, Any], list[Any]]
     ) -> Optional[SupportedColumns]:
-        if isinstance(dataset_feature, list):
-            return ListColumn(feature_name=dataset_feature_name, n_samples=num_examples, n_bins=histogram_num_bins)
-
-        if isinstance(dataset_feature, dict):
-            if dataset_feature.get("_type") == "Sequence":
+        if isinstance(dataset_feature, list) or (
+            isinstance(dataset_feature, dict) and dataset_feature.get("_type") == "Sequence"
+        ):
+            schema = pl.scan_parquet(local_parquet_glob_path).schema[dataset_feature_name]
+            if isinstance(schema, List):  # compute only if it's internally a List! because it can also be Struct
```

Comment:
add link to the docs

```suggestion
            # Compute only if it's internally a List! because it can also be Struct, see
            # https://huggingface.co/docs/datasets/v2.18.0/en/package_reference/main_classes#datasets.Features
            if isinstance(schema, List):
```